src/websocket/WebSocketClient.js

```diff
@@ -88,9 +103,11 @@ class WebSocketClient {
     this.openCount = this.openCount + 1
     this.closeCount = 0
 
-    this.options.onConnect()
+    this.options.onConnect(event)
     if (this.openCount > 1) {
-      this.options.onReconnect()
+      this.options.onReconnect(event)
+
+      JSONAsyncStorage.consume('@WsMsgQueue', (msg) => this.send(msg))
```

Comment:
I don't think this is the correct place to call this. Will it be called if you don't connect/disconnect at least once?

Reply:
The first time `open` is emitted, `@WsMsgQueue` is empty, and `consume` will not be called.

Would you prefer `consume` to be called every time `open` is emitted?

Reply:
In my mind a nice use-case, would have been not to block the interface (with a big loader) when the WS connection is not open. Before opening we don't block the user to do things so we save the message right away.
Is it clear? What do you think?

Reply:
If I understand correctly, you would prefer never to have to show any kind of "loading" spinner due to the websocket not yet being connected.

I can just move the call to `consume` out of the `if` statement, so that it will be called on every `open` event.